app/models/message.rb

```diff
@@ -37,12 +37,33 @@ class Message < ApplicationRecord
   include Liquidable
   NUMBER_OF_PERMITTED_ATTACHMENTS = 15
 
+  TEMPLATE_PARAMS_SCHEMA = {
+    'type': 'object',
+    'properties': {
+      'template_params': {
+        'type': 'object',
+        'properties': {
+          'name': { 'type': 'string' },
+          'category': { 'type': 'string' },
+          'language': { 'type': 'string' },
+          'namespace': { 'type': 'string' },
+          'processed_params': { 'type': 'object' }
+        },
+        'required': %w[name category language namespace processed_params]
+      }
+    }
+  }.to_json.freeze
```

Comment:
the schema looks good for Whatsapp template_params.  just making sure that this validation won't fail for the additional keys which we have like `campaign_id`  which can also be a part of the additional attributes

Reply:
Cool, will test this out

Reply:
@sojan-official tested this out here

https://www.loom.com/share/5e7984e43e154016a4ae4615f4b72274

Reply:
@sojan-official @scmmishra This change broke in production. The namespace key is not a mandatory one.